src/extension_utils/utils.ts

```diff
@@ -302,18 +300,24 @@ export const getTelemetryState = () => {
     .get("telemetry.enableTelemetry", true);
 };
 
-export const installPythonDependencies = async (context: vscode.ExtensionContext, pythonExecutable: string) => {
+export const installPythonDependencies = async (context: vscode.ExtensionContext, pythonExecutable: string, pathToLibs: string) => {
   let installed: boolean = false;
   try {
     vscode.window.showInformationMessage(CONSTANTS.INFO.INSTALLING_PYTHON_DEPENDENCIES);
-    const requirementsPath: string = getPathToScript(context, "out", "requirements.txt");
-    const pathToLibs: string = getPathToScript(context, "out", "python_libs");
+
+    // only use requirements file with pywin32 if on windows
+    const requirementsFileName = (process.platform === "win32") ? "requirements-windows.txt" : "requirements.txt";
```

Comment:
Do you think it would be possible to have only windows specific requirements in `requirements-windows.txt` and all common ones in `requirements.txt`? That way a developer adding/editing/removing a requirement won't need to remember to update both files.
What do you think?

Reply:
That sounds good! I'll implement that.